pkg/skaffold/schema/latest/config.go

```diff
@@ -107,6 +107,9 @@ type BuildConfig struct {
 	// These registries will be connected to via HTTP instead of HTTPS.
 	InsecureRegistries []string `yaml:"insecureRegistries,omitempty"`
 
+	// DebugHelpersRegistry is the registry from which the debug helper images are used.
+	DebugHelpersRegistry string `yaml:"debugHelpersRegistry,omitempty"`
```

Comment:
I think this might be better in the `DeployConfig` as the value is not used in the _building_ of images, but in the deployment of the images.

Reply:
You're right. It would have been better in deploy than in build, but I moved it to the global config anyway. Because as you suggested it's the better place.